plugins/Goals/Controller.php

```diff
@@ -90,6 +92,7 @@ public function goalConversionsOverview()
         $view->conversion_rate_returning = $this->formatConversionRate($goalMetrics, 'conversion_rate_returning_visit');
         $view->conversion_rate_new = $this->formatConversionRate($goalMetrics, 'conversion_rate_new_visit');
         $view->idGoal = $idGoal;
+        $view->visitorLogEnabled = Manager::getInstance()->isPluginActivated('Live') && Live::isVisitorLogEnabled($this->idSite);
```

Comment:
Can we check for `Manager::getInstance()->isPluginActivated('Live')` in `Live::isVisitorLogEnabled` or do the checks need to be separate?

Reply:
In theory guess we could do that. But might be better if we always check if a plugin is installed & activated before using it.
Not sure if Matomo would even run correctly if the Live plugin would be deleted, but if that would work, the code would break here if we directly use a part of the plugin without checking if it's available...

Reply:
👍